unstructured/documents/elements.py

```diff
@@ -143,6 +143,9 @@ class ElementMetadata:
     sent_to: Optional[List[str]] = None
     subject: Optional[str] = None
 
+    # Epub specific metadata fields
+    epub_section: Optional[str] = None
```

Comment:
Let's make this `section` instead of `epub_section` since other document types may have section information as well.